.circleci/config.yml

```
@@ -56,6 +56,30 @@
+      - image: ubuntu:latest
+        environment:
+          TZ: "Asia/Tokyo"
+    steps:
+      - checkout
+      - run:
+          name: Submodule
+          command: |
+            ./.circleci/submodule.sh
+      - restore_cache:
+          keys:
+          - v1-test-{{ checksum "package.json" }}
```

yamllint (1.35.1)

[error] wrong indentation: expected 12 but found 10

(indentation)